java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/BuildInfo.java

```diff
@@ -12,24 +12,34 @@ public class BuildInfo {
     public static final String JDK = "jdk";
     public static final String MAVEN = "mvn";
     public static final String GRADLE = "gradle";
-
     /**
      * Possible build tools, including the JDK. This is represented
      */
     Map<String, VersionRange> tools = new HashMap<>();
     /**
      * List of commands to try. Normally there is only one, but if there is both maven and gradle
-     * present then we might try invoke both.
+     * present then we might try to invoke both.
      */
     List<List<String>> invocations = new ArrayList<>();
 
     String enforceVersion;
+
     List<String> ignoredArtifacts = new ArrayList<>();
 
     public Map<String, VersionRange> getTools() {
         return tools;
     }
 
+    /**
+     * Version of the build tool.
+     */
+    String toolVersion;
```

Comment:
This info is already there, in the VersionRange of the tools map.

Reply:
I saw I could set/use the preferred version from the version range, but this variable was more about how to pass that as a parameter to the yaml file.

Reply:
We are going to need some logic in the reconciler that maps what was discovered to the builder images that are available. I am planning on looking at this once this PR is in.

Reply:
OK to leave it in for now? It currently does set `preferredVersion` and `toolVersion` to the same value.